yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -50,11 +53,13 @@ type AssessmentReport struct {
 
 	Sharding *migassessment.ShardingReport `json:"Sharding"`
 	Sizing   *migassessment.SizingReport   `json:"Sizing"`
+
+	MigrationAssessmentStats *[]migassessment.MigrationAssessmentStats `json:"MigrationAssessmentStats"`
 }
 
 type UnsupportedFeature struct {
-	FeatureName string   `json:"feature_name"`
-	ObjectNames []string `json:"object_names"`
+	FeatureName string   `json:"FeatureName"`
+	ObjectNames []string `json:"ObjectNames"`
```

Comment:
Why?

Reply:
I guess your advice was to use same casing for the keys in our json report? Am i misunderstanding something?
I was target to use CamelCase for all json tags, by replacing all existing snake_case